default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	//  - rebornix.ruby: Ruby language support and debugging for Visual Studio Code
+	//  - vscode.search-result: Provides syntax highlighting and language features for tabbed search results.
+	//  - vscode.typescript-language-features: Provides rich language support for JavaScript and TypeScript.
+	//  - aliariff.vscode-erb-beautify: Format/Beautify ERB files
+	//  - mbessey.vscode-rufo: Ruby formatter
+	//  - redhat.vscode-yaml: YAML Language Support by Red Hat, with built-in Kubernetes syntax support
+	//  - vscode.configuration-editing: Provides capabilities (advanced IntelliSense, auto-fixing) in configuration files like settings, launch, and extension recommendation files.
+	//  - vscode.debug-auto-launch: Helper for auto-attach feature when node-debug extensions are not active.
+	//  - vscode.debug-server-ready: Open URI in browser if server under debugging is ready.
+	//  - vscode.emmet: Emmet support for VS Code
+	//  - vscode.extension-editing: Provides linting capabilities for authoring extensions.
+	//  - vscode.git: Git SCM Integration
+	//  - vscode.git-base: Git static contributions and pickers.
+	//  - vscode.github: GitHub features for VS Code
+	//  - gitpod.gitpod-web: Gitpod Web Support
+	//  - vscode.grunt: Extension to add Grunt capabilities to VS Code.
+	//  - vscode.gulp: Extension to add Gulp capabilities to VSCode.
+	//  - vscode.image-preview: Provides VS Code's built-in image preview
+	//  - vscode.ipynb: Provides basic support for opening and reading Jupyter's .ipynb notebook files
+	//  - vscode.jake: Extension to add Jake capabilities to VS Code.
+	//  - ms-vscode.js-debug: An extension for debugging Node.js programs and Chrome.
+	//  - vscode.markdown-math: Adds math support to Markdown in notebooks.
+	//  - vscode.merge-conflict: Highlighting and commands for inline merge conflicts.
+	//  - vscode.microsoft-authentication: Microsoft authentication provider
+	//  - vscode.npm: Extension to add task support for npm scripts.
+	//  - vscode.simple-browser: A very basic built-in webview for displaying web content.
+	//  - vortizhe.simple-ruby-erb: Provides simple Ruby and ERB language, code snippets and ERB tag helper support for Visual Studio Code without messing with linting or debugging
+	//  - ethan-reesor.vscode-byebug: Byebug debugger support for VSCode
+	//  - ms-vscode.vscode-js-profile-table: Text visualizer for profiles taken from the JavaScript debugger
+	//  - GitHub.vscode-pull-request-github: Pull Request and Issue Provider for GitHub
+	"editor.defaultFormatter": null,
+
+	// Controls whether the Go to Definition mouse gesture always opens the peek widget.
+	"editor.definitionLinkOpensInPeek": false,
+
+	// Controls whether `editor.tabSize#` and `#editor.insertSpaces` will be automatically detected when a file is opened based on the file contents.
+	"editor.detectIndentation": true,
+
+	// Controls whether the editor should allow moving selections via drag and drop.
+	"editor.dragAndDrop": true,
```

Comment:
I think this should be `false`, it messes me up often and I almost never actually want to do this.